packages/dependency-extraction-webpack-plugin/test/fixtures/wordpress/webpack.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
 		new DependencyExtractionWebpackPlugin( {
 			requestToExternalModule( request ) {
 				if ( request.startsWith( '@wordpress/' ) ) {
-					return request;
+					return true;
```

Comment:
There are 10 other fixtures where the implementation could also be shortened to:
```js
return request.startsWith( '@wordpress/' );
```

Reply:
Do you think we should update all of them? The other style is also valid and it seems fine to test the different forms.

Reply:
It's a minor nit of course, but I'd personally update all of them. There's the `module-renames` test that tests the rename specifically, and covers the `string` return value.

Reply:
That's fair, I went ahead and updated them all.